lib/LaTeXML/Package/graphics.sty.ltxml

```diff
@@ -221,8 +221,11 @@ DefConstructor('\graphicspath DirectoryList', sub {
   properties => sub {
     my ($stomach, $paths) = @_;
     my @paths = ();
+    my $root  = $STATE->lookupValue('SOURCEDIRECTORY') || '';
     foreach my $dir ($paths->getValues) {
-      my $path = pathname_absolute(pathname_canonical(ToString($dir)));
+      $dir = pathname_canonical(ToString($dir));
+      $dir = pathname_concat($root, $dir) if $root;
```

Comment:
Isn't it a problem to unconditionally prefix `$root` if the directory already was absolute (and you're *not* in a zip)?

Reply:
You're right, I've introduced a bug when using absolute paths in the argument. My treatment should only be done in the relative path case.

Btw in general I also don't think the ZIP flow could recover an absolute path given via `\graphicspath`, so that case is likely best left as-is. I'll make a small test and retry, thanks for catching this.

Reply:
Tested and patched, exactly as you described it.